packages/binding-http/test/http-client-test.ts

```diff
@@ -443,11 +440,11 @@ class HttpClientTest2 {
         let eventSpy = chai.spy(async function (data: any) {
             eventSpy.should.have.been.called.once;
             subscribeSpy.should.have.been.called.once;
-            done();
             server.close();
+            done();
         });
 
-        server.listen(60605, "0.0.0.0");
+        server.listen(34000, "0.0.0.0");
```

Comment:
> Different operating systems (OS) use different port ranges for ephemeral ports. Many Linux versions use port range 32768-61000, while Windows versions (until XP) use 1025-5000, by default.

Aren't we again in the possible range for ephemeral ports? shouldn't it be something below 32000?